src/sql/workbench/contrib/notebook/find/notebookFindModel.ts

```diff
@@ -585,14 +584,50 @@ export class NotebookFindModel extends Disposable implements INotebookFindModel
 	// In markdown links are defined as [Link Text](https://url/of/the/text). when searching for text we shouldn't
 	// look for the values inside the (), below regex replaces that with just the Link Text.
 	cleanMarkdownLinks(cellSrc: string): string {
-		return cellSrc.replace(/(?:__|[*#])|\[(.*?)\]\(.*?\)/gm, '$1');
+		return cellSrc.replace(/(?:__)|\[([\s\S]*?)\]\(.*?\)/gm, '$1');
```

Comment:
It would be good to make these groups non-capturing since we don't need to actually store them